### ISSUE TITLE
iOS 13.0+ production Downloadfile function isn't work . but 12.0+ is work, 

### ISSUE THREAD

**Issue by FaceOnFace** [itinance/react-native-fs#815]
1. "react-native": "^0.57.4",
2."react-native-fs": "^2.13.3",

in Development iOS 13.0+ is work . in Production iOS 13.0+ isn't work, but 12.0+ is work.
How to solve? ......

**Comment by ForevereBest** [comment]
Yes, I have encountered the same problem. Is there any solution @FaceOnFace

**Comment by FaceOnFace** [comment]
![1111](https://user-images.githubusercontent.com/26252165/69856815-717d8780-12c9-11ea-9ca1-3dd316b7f6a3.png)

**Comment by Dellybro** [comment]
This is also happening to me. Any solutions?

**Comment by FaceOnFace** [comment]
> This is also happening to me. Any solutions?

Could you use Objective-C ?  source code use Objective-C download task same .  try use Objective-C download method. I used AFNetworking solve it.  maybe you  can try AFNetworking download method.
